### DISCUSSION TITLE
Student.cs Kullanımı

### DISCUSSION THREAD

**Discussion by guvenrdvan** [gusanmaz/BMB212-Algorithms-2223#13] [General]
Hocam ödev1 i klonladıktan sonra student.java 'yı klonladığımız dosya içerisine oluşturup belirttiğiniz kodları yapıştırıp derleyip repoya ekliyoruz fakat main kısmında cidden neyin nasıl kullanılacağı ya da parametreyi nasıl vereceğimiz anlaşılmıyor. Path'i nasıl kullanacağımızı da bilmiyoruz anlattığınız derste de çok karışık geldi çoğu arkadaşım da bu dertten muzdarip . Biraz açıklayabilir misiniz?

**Comment by gusanmaz** [comment]
Yol bilgisini komut satiri argumani olarak alacaksiniz ve okumus oldugunuz bu degeri degistirmeden `ReadCSV` fonksiyonuna parametre olarak vereceksiniz.

Komut satiri argumanlarinin kullanimini anlamak icin https://github.com/gusanmaz/BMB212-Algorithms-22-23#command-line-arguments baglantisindaki ornek kodu inceleyebilirsiniz. Ayrica gecen seneki dersin reposunda komut satiri argumanlarinin kullanimini gerektiren pek cok odev gorebilirsiniz. Bu odevler icin ogrencilerin gonderdigi kodlari ve aldiklari puanlari da not tablolarinda gorebilirsiniz. Tam puan alan kodlari inceleyerek komut satiri argumanlarinin nasil kullanilmasi gerektigi konusunda fikir edinebilirsiniz.
